handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -77,7 +79,7 @@ public JdkSslServerContext(File certChainFile, File keyFile, String keyPassword)
      * @param nextProtocols the application layer protocols to accept, in the order of preference.
      *                      {@code null} to disable TLS NPN/ALPN extension.
      * @param sessionCacheSize the size of the cache used for storing SSL session objects.
-     *                         {@code 0} to use the default value.
+     *                       {@code 0} to use the default value.
```

Comment:
revert. (I'm calling this out because the new indentation is clearly worse than the previous indentation.)